src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -13,7 +13,7 @@ notSupported:
 
 When a user experiences an error, Sentry provides the ability to take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
 
-This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently, this feature will be a best effort solution.
+This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be avaiable, is when the event happens before the screen starts to load. So inherently, this feature will be a best effort solution.
```

Comment:
@imatwawana I tried to add more context to why it might not work, since I imagine people will be reaching out "why don't I have a screenshot to this event". Does this sounds OK?